client/my-sites/upgrades/domain-management/style.scss

```diff
@@ -585,6 +585,116 @@ input[type=radio].domain-management-list-item__radio {
 	}
 }
 
+.domain-connect__main {
+	h2 {
+		font-size: 20px;
+		font-weight: 600;
+		margin-bottom: 1em;
+	}
+
+	.domain-connect__button {
+		margin-right: 16px;
+		margin-top: 16px;
+	}
+
+	.domain-connect__footer {
+		border-top: 1px solid $gray-light;
+		padding-top: 10px;
+	}
+
+	.domain-connect__is-placeholder {
+		span {
+			@include placeholder( 23% );
+
+			display: block;
+			margin: 0.8em 0 0.8em 0;
+			height: 1em;
+
+			&:last-child {
+				width: 30%;
+			}
+
+			&:first-child {
+				width: 100%;
+			}
+		}
+	}
+}
+
+.domain-connect__dns-list {
```

Comment:
Well, I suppose I could extract the styles for `dns_list` into a mixin and then `@include` it in both places. (The linter won't let me use `dns__list` in my `className`s.)